CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.2.0
```

Comment:
should the pom.xml file version also match this version?

Reply:
I updated the pom to match the version